packages/xstate-react/test/createActorContext.test.tsx

```diff
@@ -326,4 +326,114 @@ describe('createActorContext', () => {
       expect(screen.getByTestId('value').textContent).toBe('42');
     });
   });
+
+  it('should expect a constant machine reference', () => {
```

Comment:
this seems to be tested indirectly, could we either adjust the test case (by using more targeted assertions) or adjust this test title somehow?

Reply:
Open to suggestions here

Reply:
```suggestion
  it("should preserve machine's identity when swapping options using in-render `.provide`", () => {
```